.github/workflows/main.yml

```diff
@@ -20,52 +22,23 @@ jobs:
       uses: microsoft/setup-msbuild@v1
 
     - name: Restore
-      run: msbuild /restore Xamarin.Forms.sln
-    
+      run: |
+        msbuild /restore Xamarin.Forms.sln
+
```

Comment:
nit: looks like this 3-line hunk above is not needed?

Reply:
:+1:

Reply:
if you have addressed this, then click on "Resolve conversation" please